### DISCUSSION TITLE
Vuexy Dashboard Template

### DISCUSSION THREAD

**Discussion by optimbiz** [quasarframework/quasar#15433] [CLI - Capacitor / Cordova modes]
I'm interested in doing a project on Quasar, PWA and mobile apps, while using a VUE3 ready made tempalte: https://pixinvent.com/demo/vuexy-vuejs-admin-dashboard-template/landing/

Would it be possible to somehow import the files or do we need to rewrite everything on Quasar Framework CSS, component standard styles ?

**Comment by corepay** [comment]
;) I purchased Vuexy several weeks ago myself.  AFAIK you need to replace the vuetify components in the template with the Quasar counterpart. 

IMO Use Vuexy as a guide and rewrite each page or component from scratch...

**Comment by pratik227** [comment]
You can checkout https://quasar-admin-premium.netlify.app
